experimenter/experimenter/experiments/changelog_utils.py

```diff
@@ -167,6 +168,7 @@ def get_formatted_change_object(field_name, field_diff, changelog, timestamp):
         )
 
     return {
+        "id": str(uuid.uuid4()),
```

Comment:
Why this change? And why is it a different UUID for each call?

Reply:
Yeah so I needed to have some unique identifier for each record to use as the html ID for the 'show more' id reference, and I couldn't use the db id since a single changelog appears as multiple entries in the history page.  I'm open to suggestions about some other way to uniquely identify each row?

Reply:
I'm probably missing something important here, but if I'm correctly understanding what this is doing, could you do something like `${changelog.id}_${field_name}`?

Reply:
Yeah I was thinking that too but then we're placing constraints on those fields, when really we just want something that says 'this is an arbitrary unique identifier for this row.'  Another way to do it is to hash its contents.  I'm not sure any one is preferable over another?